src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/AC8085.scala

```diff
@@ -18,16 +18,17 @@ package uk.gov.hmrc.ct.accounts.frs102.boxes
 
 import uk.gov.hmrc.ct.accounts.frs102.retriever.Frs102AccountsBoxRetriever
 import uk.gov.hmrc.ct.box._
+import uk.gov.hmrc.ct.box.retriever.FilingAttributesBoxValueRetriever
 
 case class AC8085(value: Option[Boolean]) extends CtBoxIdentifier(name = "These accounts have been prepared in accordance with the provisions applicable to companies subject to the small companies regime.")
   with CtOptionalBoolean
   with Input
-  with ValidatableBox[Frs102AccountsBoxRetriever]
+  with ValidatableBox[Frs102AccountsBoxRetriever with FilingAttributesBoxValueRetriever]
   with Validators {
 
-  override def validate(boxRetriever: Frs102AccountsBoxRetriever): Set[CtValidation] = {
+  override def validate(boxRetriever: Frs102AccountsBoxRetriever with FilingAttributesBoxValueRetriever): Set[CtValidation] = {
     collectErrors(
-      validateAsMandatory(this)
+      failIf(boxRetriever.abridgedFiling().value || boxRetriever.statutoryAccountsFiling().value)(validateAsMandatory(this))
```

Comment:
You dont need the flag conditions here either (its always going to be one or the other ...)